libr/Makefile

```diff
@@ -42,7 +40,30 @@ libr.${EXT_SO}:
 	rm -rf .libr/tcc # WHY
 	$(CC) $(LDFLAGS) -shared -dynamic -arch arm64 -o libr.${EXT_SO} .libr/*/*.o ../shlr/gdb/lib/libgdbr.a
 
-include Makefile-ar.mk
+STATIC_LIBS:=$(wildcard */libr_*.${EXT_AR}) $(shell find ../shlr -name '*.${EXT_AR}')
+libr.${EXT_AR}: ${STATIC_LIBS}
+	rm -rf .libr
+	mkdir .libr
+	cd .libr ; \
+		for FILE in ${STATIC_LIBS} ; do \
+			BASE=$$(basename $${FILE} .a) ; \
+			PREVOBJ=none ; \
+			for OBJ in $$(ar t ../$${FILE} | sort) ; do \
+				if [ $${PREVOBJ} = $${OBJ} ] ; then \
+					N=$$(expr $${N} + 1) ; \
+				else \
+					N=1 ; \
+				fi ; \
+				ar xN $${N} ../$${FILE} $${OBJ} ; \
```

Comment:
use $(AR) instead of `ar`

Reply:
xN flag is a gnuism, doesnt works on BSD or Mac ar tool, so its wrong

Reply:
Bleh, too bad. Let me redo this one more time..